lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -315,6 +315,16 @@ def initialize(model, current_user:, embed_links: false)
           end
         end
 
+        links :additionalWorkPackageTabs,
+              uncacheable: true do
+          additional_work_package_tabs.map do |tab_identifier|
+            {
+              href: "#{work_package_path(id: represented.id)}/additionalTabs/#{tab_identifier}",
```

Comment:
this seems weird to me - we don't really need this `href`, but I need it to make a proper link.